modules/serverless-ossm-secret-filtering-net-istio.adoc

```diff
@@ -22,10 +24,8 @@ If you enable secret filtering, all of your secrets need to be labeled with  `ne
 * Install the {ServerlessOperatorName} and Knative Serving.
 * Install the OpenShift CLI (`oc`).
 
-.Procedure
+You can disable the secret filtering by setting `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID=false` via `workloads` in the `KnativeServing` custom resource (CR).
```

Comment:
```suggestion
You can disable the secret filtering by setting the `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID` variable to `false` in the `KnativeServing` custom resource (CR).
```

Reply:
I think specifying the name of the variable (`ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID`) and CR would suffice here, since the example configuration is shown below.